redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
+                "LURKABLE": _("Lurkable"),
+                "NEWS": _("News Channels"),
+                "ANIMATED_ICON": _("Animated Icon"),
+                "BANNER": _("Banner Image"),
+                "MEMBER_LIST_DISABLED": _("Member list disabled"),
+            }
+            guild_features_list = [
+                f"✅ {name}" for feature, name in features.items() if feature in set(guild.features)
+            ]
+
+            joined_on = _(
+                "{bot_name} joined this server on {bot_join}. That's over {since_join} days ago!"
+            ).format(
+                bot_name=ctx.bot.user.name,
+                bot_join=guild.me.joined_at.strftime("%d %b %Y %H:%M:%S"),
+                since_join=humanize_number((ctx.message.created_at - guild.me.joined_at).days),
+            )
+
+            data = discord.Embed(description=created_at, colour=await ctx.embed_colour())
+            data.set_author(
+                name=guild.name,
+                icon_url="https://cdn.discordapp.com/emojis/457879292152381443.png"
+                if "VERIFIED" in guild.features
+                else "https://cdn.discordapp.com/emojis/508929941610430464.png"
+                if "PARTNERED" in guild.features
+                else discord.Embed.Empty,
+            )
+            data.set_thumbnail(
+                url=guild.icon_url
+                if guild.icon_url
+                else "https://cdn.discordapp.com/embed/avatars/1.png"
+            )
+            data.add_field(name=_("Members:"), value=member_msg)
+            data.add_field(
+                name=_("Channels:"),
+                value=_(
+                    "\N{SPEECH BALLOON} Text: {text}\n"
+                    "\N{SPEAKER WITH THREE SOUND WAVES} Voice: {voice}"
+                ).format(text=bold(text_channels), voice=bold(voice_channels),),
+            )
+            data.add_field(
+                name=_("Utility:"),
+                value=_(
+                    "Owner: {owner}\nRegion: {region}\nVerif. level: {verif}\nServer ID: {id}"
+                ).format(
+                    owner=bold(str(guild.owner)),
+                    region=bold(vc_regions[str(guild.region)]),
+                    verif=bold(verif[str(guild.verification_level)]),
+                    id=bold(str(guild.id)),
+                ),
+                inline=False,
+            )
+            data.add_field(
+                name=_("Misc:"),
+                value=_(
+                    "AFK channel: {afk_chan}\nAFK Timeout: {afk_timeout}\nCustom emojis: {emojis}\nRoles: {roles}"
+                ).format(
+                    afk_chan=bold(str(guild.afk_channel))
+                    if guild.afk_channel
+                    else bold(_("Not set")),
+                    afk_timeout=bold(humanize_timedelta(seconds=guild.afk_timeout)),
+                    emojis=bold(humanize_number(len(guild.emojis))),
+                    roles=bold(humanize_number(len(guild.roles))),
```

Comment:
Nitpick, small i18n improvement.
```suggestion
                    "AFK channel: {afk_chan}\nAFK Timeout: {afk_timeout}\nCustom emojis: {emoji_count}\nRoles: {role_count}"
                ).format(
                    afk_chan=bold(str(guild.afk_channel))
                    if guild.afk_channel
                    else bold(_("Not set")),
                    afk_timeout=bold(humanize_timedelta(seconds=guild.afk_timeout)),
                    emoji_count=bold(humanize_number(len(guild.emojis))),
                    role_count=bold(humanize_number(len(guild.roles))),
```